packages/adapters/eigenda/src/connector.rs

```diff
@@ -144,29 +161,18 @@ impl<S> EigenDAClient<S> {
                 .expect("TODO: segfault"),
         );
 
-        // Placeholder Service Manager Address - this should be configured properly for production
-        let svc_manager_address = H160::from_str("d4a7e1bd8015057293f0d0a557088c286942e84b")
-            .map_err(|e| Error::Other(format!("Invalid service manager address: {e}")))?;
-
-        // Default SRS points URLs from Eigen-DA repos
-        let srs_g1_url = "https://github.com/Layr-Labs/eigenda-proxy/raw/2fd70b99ef5bf137d7bbca3461cf9e1f2c899451/resources/g1.point".to_string();
-        let srs_g2_url = "https://github.com/Layr-Labs/eigenda-proxy/raw/2fd70b99ef5bf137d7bbca3461cf9e1f2c899451/resources/g2.point.powerOf2".to_string();
-
-        let config = EigenConfig::new(
-            disperser_rpc_url,
-            eth_rpc_url,
-            8, // Set confirmation depth as needed
-            svc_manager_address,
-            false, // Set true to wait for Ethereum finalization
-            true,  // Set true if using authenticated disperser endpoints
-            SrsPointsSource::Url((srs_g1_url, srs_g2_url)),
-            vec![], // Specify quorum IDs if using custom quorums
-        )
-        .map_err(|e| Error::Other(format!("Failed to create EigenConfig: {e}")))?;
-
-        // Create EigenClient instance
-        let blob_provider = Arc::new(DummyBlobProvider);
-        let eigen_client = EigenClient::new(config, signer, blob_provider)
+        let config = PayloadDisperserConfig {
+            polynomial_form: PayloadForm::Coeff,
+            blob_version: 0,
+            cert_verifier_address: H160::from_str("d4a7e1bd8015057293f0d0a557088c286942e84b")
```

Comment:
magic number. Can we make this into a const?

Reply:
addressed in 69943ea
this will have some changes later when i make it configurable